404.html

```diff
@@ -1,266 +1,252 @@
-			type="image/svg+xml"
-		/>
-		<title>If These Lands Could Talk - 404 Page Not Found</title>
-		<!-- google fonts -->
-		<link rel="preconnect" href="https://fonts.googleapis.com" />
-		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-		<link
-			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
-			rel="stylesheet"
-		/>
-		<!-- styles -->
-		<link rel="stylesheet" href="css/global.css" />
-		<link rel="stylesheet" href="css/404.css" />
-	</head>
 
-	<body>
-		<!-- Header Navigation-->
-		<header>
-			<!-- Navigation -->
-			<nav>
-				<div class="wrapper">
-					<div class="flex-container">
-						<!-- ITLCT Logo -->
-						<a
-							href="/index.html"
-							class="img-link"
-							aria-label="If These Lands Could Talk Home Page Link"
-						>
-							<img
-								src="/assets/logos/itlct-logo.svg"
-								alt="If These Lands Could Talk Logo"
-							/>
-						</a>
-						<!-- Navigation Menu -->
-						<div class="mobile-menu-container">
-							<!-- mobile menu button -->
-							<button
-								id="mobile-menu"
-								class="mobile-menu"
-								aria-label="toggle mobile menu on"
-							>
-								<!-- Hamburger Icon -->
-								<img
-									src="/assets/icons/global/menu-icon.svg"
-									alt="mobile menu icon"
-									class="menu-icon"
-								/>
-								<!-- exit menu icon -->
-								<img
-									src="/assets/icons/global/close-icon.svg"
-									alt="exit mobile menu icon"
-									class="menu-exit-icon hidden"
-								/>
-							</button>
-
-							<!-- Page Links -->
-							<ul id="nav-links" class="nav-links">
-								<li><a href="/about.html">About Us</a></li>
-								<li><a href="/impact.html">Our Impact</a></li>
-								<li><a href="/values.html">Our Values</a></li>
-								<li><a href="/events.html">Events</a></li>
-								<li><a href="/team.html">Our Team</a></li>
-								<li><a href="/contact.html">Contact</a></li>
-								<!-- Mobile Support Button -->
-								<li class="mobile-support-btn"><a href="#">Support Us</a></li>
-							</ul>
-						</div>
-
-						<!-- desktop support btn -->
-						<a href="#" class="btn-main desktop-support-btn">Support Us</a>
-					</div>
-				</div>
-			</nav>
-		</header>
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<link
+		rel="shortcut icon"
+		href="assets/logos/favicon.svg"
+		type="image/svg+xml" />
+	<title>If These Lands Could Talk - 404 Page Not Found</title>
+	<!-- google fonts -->
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+		rel="stylesheet" />
+	<!-- styles -->
+	<link rel="stylesheet" href="css/global.css" />
+	<link rel="stylesheet" href="css/404.css" />
+	<!-- MailerLite Universal -->
+	<script>
```

Comment:
looks good - can you please add this to **main.js** or another **component.js** file? I think it is easier to maintain this way.